.env.test

```diff
@@ -0,0 +1 @@
+POSTGRES_PRISMA_URL="postgresql://postgres:password@localhost:5432/court-share?schema=public"
```

Comment:
This is used locally, it swaps the .env with this file to connect into the test database.